static/js/controllers/inbox.js

```diff
@@ -148,6 +148,16 @@ require('moment/locales');
         $scope.items = contacts || [];
       };
 
+      $scope.removeContact = function(contact) {
```

Comment:
Ref renaming `*DeletedContacts` to `*DeletedMessages`, should I also look at why the messages are stored in a variable called `options.contacts`?!